.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: golang-1.14
+  tag: golang-1.16
```

Comment:
I think we should land https://github.com/openshift/baremetal-operator/pull/177/ instead then rebase, right?